tests/test_suites/ShortSystemTests/test_many_parallel_reads.sh

```diff
@@ -0,0 +1,67 @@
+timeout_set 2 minutes
+
+CHUNKSERVERS=12 \
+    USE_RAMDISK=YES \
+    MOUNT_EXTRA_CONFIG="readaheadmaxwindowsize=256,maxreadaheadrequests=30,readworkers=100" \
+    MASTER_CUSTOM_GOALS="8 ec_8_4: \$ec(8,4)"
+    setup_local_empty_saunafs info
```

Comment:
Perhaps cacheexpirationtime should be set to something else to make sure it's on (as it is currently 0)?

Reply:
Oh wait, sorry, I didn't see you set it back.

Reply:
Branch was not up to date with dev until now, so it was having the default cacheexpirationtime of 1000. And, now the branch brought back the 1000 default value. I guess it is fine.